discord/gateway.py

```diff
@@ -553,6 +571,15 @@ def from_client(cls, client):
         }
 
         yield from ws.send_as_json(identify)
+
+        try:
+            # Poll for useless HELLO event, then READY
+            yield from asyncio.wait_for(ws.poll_event(), timeout=60, loop=client.loop)
```

Comment:
this is hella error prone. ur first 2 messages are not guaranteed to be HELLO then READY. you can get a heartbeat request in-between them very feasibly.